modules/apps/data-engine/data-engine-test/src/testFunctional/tests/macros/DataEngineUpgrade.macro

```diff
@@ -37,9 +37,79 @@ definition {
 
 	macro editAnUpgradedWebContentWithTranslations {
 
-		// TODO LPS-126237 editAnUpgradedWebContentWithTranslations pending implementation
-
-		echo("empty stub");
+		ProductMenu.gotoPortlet(
```

Comment:
Hey @Odrakir1 , we can use `WebContentNavigator.openWebContentAdmin(siteURLKey = "guest");` to navigate to WEM Admin without using `ProductMenu.gotoPortlet()` which navigates via clicks and have asserts in it.